app/services/idv/steps/temp_maybe_redirect_to_verify_info_helper.rb

```diff
@@ -1,17 +1,22 @@
+##
 # This module and calls to it can be removed when the in_person_verify_info_controller_enabled
 # flag is removed.
 #
-
 module Idv
   module Steps
     module TempMaybeRedirectToVerifyInfoHelper
       private
 
-      def maybe_redirect_to_verify_info
+      def maybe_redirect_to_verify_info(skip = false)
         return unless IdentityConfig.store.in_person_verify_info_controller_enabled
+        return if skip
         flow_session[:flow_path] = @flow.flow_path
         redirect_to idv_in_person_verify_info_url
       end
+
+      def redirect_to_address
+        redirect_to idv_address_path
```

Comment:
I've generally seen redirects with `_url` instead of `_path`.

Reply:
Deleted with commit e46ba6502eb69fb8817ebff7e4da767340b878d3
 It was not being used. I forgot to remove it